server/src/main/java/io/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -84,25 +82,13 @@ public ClientQuerySegmentWalker(
     final FinalizeResultsQueryRunner<T> baseRunner = new FinalizeResultsQueryRunner<T>(
         toolChest.postMergeQueryDecoration(
             toolChest.mergeResults(
-                new MetricsEmittingQueryRunner<T>(
```

Comment:
@xvrl sorry to keep you waiting, i have been away.
anyways, it used to report pretty much same metric as the request/time from QueryResource. Instead we moved it to IntervalChunkingQueryRunner which now reports query/time for each chunk (if chunking is used).